Foundation/NSJSONSerialization.swift

```diff
@@ -240,6 +242,14 @@ private struct JSONWriter {
     let pretty: Bool
     let writer: (String?) -> Void
     
+    private lazy var _numberformatter: CFNumberFormatter = {
+        let formatter: CFNumberFormatter
+        formatter = CFNumberFormatterCreate(nil, CFLocaleCopyCurrent(), kCFNumberFormatterNoStyle)
+        CFNumberFormatterSetProperty(formatter, kCFNumberFormatterMaxFractionDigits, 15._bridgeToObject())
```

Comment:
The syntax for bridging has changed; I would suggest to actually just create a NSNumber via `NSNumber(value: 15)`